packages/blob/src/index.node.test.ts

```diff
@@ -481,4 +489,130 @@ describe('blob client', () => {
       ).rejects.toThrow(new Error('Vercel Blob: access must be "public"'));
     });
   });
+
+  describe('client put', () => {
+    beforeEach(() => {
+      process.env.BLOB_READ_WRITE_TOKEN = undefined;
+      mockClient = mockAgent.get(BASE_URL);
+      jest.resetAllMocks();
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-explicit-any
+      global.window = {} as any;
+    });
+    afterEach(() => {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-explicit-any
+      global.window = undefined as any;
```

Comment:
I wonder if we should run this test in jsdom Jest mode instead of relying on the node env of Jest. Thoughts?